README.md

```diff
@@ -0,0 +1,41 @@
+# Ahoy::Intercom
+
+## Installation
+
+Add this line to your application's Gemfile:
+
+```ruby
+gem 'ahoy-intercom'
+```
+
+And then execute:
+
+    $ bundle
+
+Or install it yourself as:
+
+    $ gem install ahoy-intercom
+
+## Usage
+
+Set your environment variable `ENV['INTERCOM_ACCESS_TOKEN']`.
```

Comment:
Environment variable is `INTERCOM_ACCESS_TOKEN`, not `ENV['INTERCOM_ACCESS_TOKEN']`